lib/Array.fx

```diff
@@ -158,6 +178,11 @@ operator * (a: 't [,], b: 't [,])
     c
 }
 
+operator *= (a: 't [,], b: 't [,]) {
+    val intermediate = a*b
+    for el@idx <- intermediate {a[idx] = el}
```

Comment:
1. normally a shorter `temp` is used instead of `intermediate`
2. if `intermediate` has smaller size than `a` than only a part of `a` will be modified, which is a bug. A better (and faster) solution is to use `a[:,:] = intermediate` instead of the loop. Such assignment operation will automatically check that the left and the right sides have the same size.

Reply:
Done.